src/components/InstrumentConfigForm.vue

```diff
@@ -316,14 +316,6 @@ export default {
         } else {
           this.instrumentConfig.mode = 'fowler16_coadds1';
         }
-      } else if(this.selectedInstrument === 'BLANCO_NEWFIRM') {
```

Comment:
They wanted to switch to using extra_params for fowler and coadds instead of the readout mode, so this auto-setting code is no longer relevant.